solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -1149,41 +1137,41 @@ contract LiquidityProtection is ILiquidityProtection, Utils, Owned, ReentrancyGu
     }
 
     /**
-     * @dev returns a protected liquidity from the store
+     * @dev returns a position from the store
      *
-     * @param id protected liquidity id
+     * @param id position id
      *
-     * @return protected liquidity
+     * @return a position
      */
-    function protectedLiquidity(uint256 id) internal view returns (ProtectedLiquidity memory) {
-        ProtectedLiquidity memory liquidity;
+    function position(uint256 id) internal view returns (Position memory) {
```

Comment:
Would have named it `get_position` or else. I think it would be a good thing for us to start having conventions for view/pure functions (if we do have i'm not aware).  i.e appending/prepending them with something particular

Reply:
We have actually discussed this in the past and decided to avoid `get` prefixes as much as possible in Solidity (for example, since this convention was in conflict with the default getters).